recipes/geant4/cross-linux.cmake

```diff
@@ -0,0 +1,20 @@
+# this one is important
```

Comment:
Is this file actually needed?

Reply:
According to the [conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#an-aside-on-cmake-and-sysroots), yes.